src/components/thumbnail-card/ThumbnailCard.js

```diff
@@ -9,13 +11,37 @@ import './ThumbnailCard.scss';
 type Props = {
     className?: string,
     icon?: React.Node,
+    id?: number,
+    setIsCardInFocus?: (isInFocus: number) => void,
+    shadowOnHover?: boolean,
     subtitle?: React.Node,
     thumbnail: React.Node,
     title: React.Node,
 };
 
-const ThumbnailCard = ({ className = '', icon, subtitle, title, thumbnail, ...rest }: Props) => (
-    <div className={`thumbnail-card ${className}`} {...rest}>
+const ThumbnailCard = ({
+    className = '',
+    icon,
+    id,
+    shadowOnHover = false,
+    setIsCardInFocus = noop,
+    subtitle,
+    title,
+    thumbnail,
+    ...rest
+}: Props) => (
+    <div
+        onFocus={() => id && setIsCardInFocus(id)}
```

Comment:
Can you explain why onFocus and onBlur were needed? This code seems more about making the example work.

it should be possible to implement this using `tabIndex` and `:active` on the relevant elements in the component and css respectively.

Reply:
Good call out! Discussed offline that we will remove this functionality - it was originally added due to the actionItem use case.